File: refoctor/src/main/java/com/est7/refoctor/影片出租/pre/Rental.java
package com.est7.refoctor.影片出租.pre;

/**
 * 租赁
 * @author: est7
 * @date: 2023/3/6
 */
public class Rental {
    private Movie _movie; // 影片
    private int _daysRented; // 租期

    public Rental(Movie movie, int daysRented) {
        _movie = movie;
        _daysRented = daysRented;
    }

    public int get_daysRented() {
        return _daysRented;
    }

    public Movie get_movie() {
        return _movie;
    }
}

File: refoctor/src/main/java/com/est7/refoctor/影片出租/refacted-01/Rental.kt
package com.est7.refoctor.影片出租.`refacted-01`

/**
 * 租赁
 * @author: est7
 * @date: 2023/3/6
 */
class Rental(// 影片
    private val _movie: Movie, // 租期
    private val _daysRented: Int
) {
    fun get_daysRented(): Int {
        return _daysRented
    }

    fun get_movie(): Movie {
        return _movie
    }
}